com.locomalito.abbayedesmorts.json

```diff
@@ -0,0 +1,42 @@
+{
+  "app-id": "com.locomalito.abbayedesmorts",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "18.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "abbayev2",
+  "finish-args": [
+      "--device=dri",
+      "--share=ipc",
+      "--socket=pulseaudio",
+      "--socket=wayland",
+      "--socket=x11"
+  ],
+  "modules": [
+      {
+          "name": "abbayedesmorts",
+          "buildsystem": "simple",
+          "build-commands": [
+              "make PREFIX=/app",
+              "make install PREFIX=/app",
+              "install -Dm644 com.locomalito.abbayedesmorts.desktop /app/share/applications/com.locomalito.abbayedesmorts.desktop",
+              "install -Dm644 com.locomalito.abbayedesmorts.appdata.xml /app/share/metainfo/com.locomalito.abbayedesmorts.appdata.xml"
```

Comment:
```suggestion
              "install -Dm644 com.locomalito.abbayedesmorts.metainfo.xml /app/share/metainfo/com.locomalito.abbayedesmorts.metainfo.xml"
```

Reply:
fixed in 0bab1b7cc2b73133fc013f36453fc031bf989819 and e18979c6e6964f793ec08865c93ae7b80502e586